charts/opentelemetry-demo/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.6.2
+version: 0.6.3
```

```suggestion
version: 0.7.0
```
Normally we do appVersion bumps as a minor bump.

---

You'll have to regenerate the examples.